README.md

```diff
@@ -16,31 +16,36 @@ Install Git LFS here: [https://git-lfs.com/](https://git-lfs.com/).
 
 ## SDK Plugin Usage
 
-The SDK supports most Hathora Cloud API endpoints, which you can [call manually](#manually-calling-api-endpoints). The SDK also provides an easy interface to automatically fetch and ping all available Hathora Cloud regions with the [Get Regional Pings](#get-regional-pings) function.
+The SDK supports most Hathora Cloud API endpoints, which you can [call manually](#manually-calling-api-endpoints). The SDK also provides an easy interface to automatically fetch and ping all available Hathora Cloud regions with the [Get Pings For Regions](#get-pings-for-regions) function.
 
-### Get Regional Pings
+### Get Pings For Regions
 
-This function takes an optional second argument `int32 NumPingsPerRegion` which defaults to `3`. The SDK will execute the following when calling this function:
-1. Call the `GetPingServiceEndpoints` API call
-2. Send an ICMP echo for each region once in parallel
-3. After receiving a result for **each** region, step 2 is repeated `NumPingsPerRegion - 1` times. The SDK waits until a result is returned from each region before sending another set of pings to prevent some of the ICMP echos from being ignored/dropped due to subsequent/overlap requests.
-4. After running step 2 a total of `NumPingsPerRegion` times, the minimum is returned for each region. If a particular region timed out or was unreachable otherwise for each ping, it is omitted from the callback delegate.
+This function takes 4 arguments:
+1. A `TMap<FString, FString>` which maps regions names (whatever you'd like to call them) to URLs
+    - The URLs must include a `:<port>` suffix if using UDPEcho ping type; you can include this suffix for ICMP ping type calls too. Hathora servers support ports `443` and `10000` for the UDP echo requests
+    - It's recommended for your backend/matchmaker to provide this map of supported regions; there is a helper function `UHathoraSDK::GetRegionMap()` which provides a hardcoded map of all the regions.
+1. A `EHathoraPingType PingType` of either `EHathoraPingType::ICMP` or `EHathoraPingType::UDPEcho`. Both send an ICMP packet; the former uses the ICMP protocol and the latter uses the UDP protocol (expecting an echo response)
+1. A callback with the results
+1. An optional `int32 NumPingsPerRegion` which defaults to `3`
```

Comment:
Minor: Looks like the numbers got malformed here (all of them are `1.` instead of incrementing)

Reply:
This is markdown syntax which renders a number list in a descending order.

Reply:
@jchu231 Can I merge without changing these? As Jared mentioned, this syntax is automatically rendered properly; it's useful so that if you insert a new step or move steps, you don't have to go back and fix all the numbers. You can see it rendered here: https://github.com/hathora/hathora-unreal-sdk/tree/ms/decouple-api-usage
![image](https://github.com/user-attachments/assets/6a1e7597-1e3d-4143-b383-5a5659522b95)